tests/x11regressions/gnomecase/application_starts_on_login.pm

```diff
@@ -104,7 +99,8 @@ sub run {
     wait_still_screen;
     send_key "alt-f4";
 
-    logout_and_login;
+    handle_logout;
+    handle_login;
```

Comment:
Is `logout_and_login` still used then? Maybe do `handle_logout` and `handle_login` in `logout_and_login`?

Reply:
The subroutine in application_starts_on_login.pm was totally dropped.

There is a logout_and_login subroutine in change_password.pm, but it will input the changed password when logging in.